src/bp/ui-studio/src/web/components/Layout/index.tsx

```diff
@@ -120,7 +120,7 @@ const Layout: FC<Props> = (props: Props) => {
     e.preventDefault()
 
     if (props.docHints.length) {
-      window.open(`https://botpress.com/docs/${props.docHints[0]}`, '_blank')
+      window.open('https://botpress.com/docs/build/dialogs#flows', '_blank')
```

Comment:
That's not the proper fix. The help button is contextual to what page you are currently on, this means that everywhere in the application you will always end up on the dialog help page, even when you are on the CMS